rust/android/Cargo.toml

```diff
@@ -8,22 +8,12 @@ edition = "2021"
 ci = []
 regchest = []
 
-[dependencies]
-jni = { version = "0.19.0", default-features = false }
-android_logger = "0.11"
-log = { workspace = true }
-rustlib = { path = "../lib" }
-
 [dev-dependencies]
-tokio =  { version = "1.24.2", features = ["full"] }
 zingoconfig = { workspace = true }
+zingomobile_utils = { path = "../zingomobile_utils" }
+regchest_utils = { git="https://github.com/zingolabs/zingo-regchest", default-features=true, branch = "dev" }
 zingo-testutils = { git="https://github.com/zingolabs/zingolib", default-features=true, branch = "stable" }
 darkside-tests = { git="https://github.com/zingolabs/zingolib", default-features=true, branch = "stable" }
+tokio =  { version = "1.24.2", features = ["full"] }
```

Comment:
I propose that we use: `tokio = { version = "1.24", features = ["full"] }`

What do you think @juanky201271 @AloeareV ?

Reply:
I can change this anytime...

Reply:
agreed
```suggestion
tokio =  { version = "1.24", features = ["full"] }
```